memdown.js

```diff
@@ -2,12 +2,12 @@ var inherits          = require('inherits')
   , AbstractLevelDOWN = require('abstract-leveldown').AbstractLevelDOWN
   , AbstractIterator  = require('abstract-leveldown').AbstractIterator
   , ltgt              = require('ltgt')
-  , setImmediate      = global.setImmediate || process.nextTick
+  , setImmediate      = require('immediate')
   , createRBT = require('functional-red-black-tree')
   , globalStore       = {}
 
 function toKey (key) {
-  return typeof key == 'string' ? '$' + key : JSON.stringify(key)
+  return '$' + key
```

Comment:
ah, ok understood. then let's rename this function, so it's more clear for the next reader? what about something like `toLocation`?